hs/Css/Loading.hs

```diff
@@ -0,0 +1,45 @@
+{-# LANGUAGE OverloadedStrings #-}
+
+module Css.Loading
+    (loadingStyles) where
+
+import Clay
+import Prelude hiding ((**))
+import Data.Monoid
+import Css.Constants
+
+{- loadingStyles
+ - Generates all CSS for the draw page. -}
+loadingStyles :: Css
+loadingStyles = do
+    spinnerCSS
+    smallCompassPosCSS
+    largeCompassPosCSS
+    spinCSS
+
+{- Properties of rotation animation. -}
```

Comment:
I copied `css/draw.hs` which used that style for commenting, and made my changes.